xbmc/platform/posix/filesystem/SMBWSDiscovery.cpp

```diff
@@ -102,7 +103,7 @@ void CWSDiscoveryPosix::SetItems(std::vector<wsd_req_info> entries)
 {
   {
     CSingleLock lock(m_critWSD);
-    m_vecWSDInfo = entries;
+    m_vecWSDInfo = std::move(entries);
   }
 }
 } // namespace WSDiscovery
```

Comment:
out of curiosity, why not pass `const std::vector<wsd_req_info>& entries` and keep `m_vecWSDInfo = entries;`?
because binding the reference would be an extra step or is there a different reason?